ironfish/src/rpc/server.ts

```diff
@@ -73,4 +76,21 @@ export class RpcServer {
       this._startPromise = promise
     }
   }
+
+  /** Authenticate the RPC request */
+  authenticate(requestAuthToken: string | undefined): boolean {
+    if (requestAuthToken) {
+      this.logger.debug(`Missing Auth token in RPC request.`)
+      return false
+    }
+
+    const rpcAuthToken = this.node.internal.get('rpcAuthToken')
```

Comment:
Here's a security question, but should we let the token change after the server starts, or should we cache this when the server starts and require you to restart the RpcServer to pick up the new token?